lib/features/home/content_panel/widgets/diary_header.dart

```diff
@@ -5,7 +5,7 @@ class _DiaryHeader extends StatelessWidget {
   final String displayName;
   final DateTime dateTime;
 
-  final String dateFormat = 'MMM dd, yyyy';
+  final String dateFormat = 'dd MMM, yyyy';
```

Comment:
why do we have 2 diary_header.dart? I think the layout are the same right? We just need 1 right?